elk/utils_evaluation/ccs.py

```diff
@@ -80,25 +62,24 @@ def get_loss(self, p0, p1):
 
         return consistency_loss + confidence_loss
 
-    # return the probability tuple (p0, p1)
-
-    def transform(self, data: list, theta_np=None):
-        if theta_np is None:
-            theta_np = self.best_theta
-        z0 = torch.tensor(self.add_ones_dimension(data[0]).dot(theta_np.T))
-        z1 = torch.tensor(self.add_ones_dimension(data[1]).dot(theta_np.T))
```

Comment:
Similarly, maybe we should keep add_ones_dimension for now. Just for reproducing the actual results of the paper. It could be of course that it doesn't make any difference, and we remove it later in a different pull request.

Reply:
Done (in the probe with the original init).